README.md

```diff
@@ -1,14 +1,16 @@
 # GlobWalk #
-A cross platform crate for recursively walking over paths matching a Glob pattern.
+Recursively find files in a directory using globs.
 
-Based on both `walkdir` &️ `globset` (❤), this crate inherits many goodies from both, such as limiting search depth and amount of open file descriptors. 
+Based on both `walkdir` &️ `globset` (❤), this crate inherits many goodies from
```

Comment:
Is this still true? Also I can't see whatever unicode symbol you have after `walkdir` on macos 10.12

Reply:
It technically is, since ignore uses globset, but I already changed it locally to ignore.

The symbol after globset is a heart emoji, there's nothing non-ascii otherwise AFAICT

Reply:
> The symbol after globset is a heart emoji, there's nothing non-ascii otherwise AFAICT

The heart is just fine, I meant the other one:

<img width="211" alt="bildschirmfoto 2018-04-20 um 11 12 28" src="https://user-images.githubusercontent.com/20063/39042698-c25f4c8c-448b-11e8-8e61-6112949fd04e.png">

Reply:
Weird, that's an ampersand, I'll make sure it's actually an ampersand and not some weird unicode abomination.